### ISSUE TITLE
3rd Learning

### ISSUE THREAD

**Issue by roqhdehd502** [roqhdehd502/Franklin-Detecter#4]
### 진행 전
- 기존 CNN 구조(NaminNeuralNetwork)에서 은닉층 2개와 배치 정규화 추가
![NaminNeuralNetwork_mk2](https://user-images.githubusercontent.com/47406388/68990119-a5e95080-0892-11ea-97b9-ee51d69444c1.jpg)
### 진행 중
- Tensorflow, numpy, matplotlib Import
- 이미지 파일을 불러온 후 텍스트 파일과 통해 이미지 전처리
- CNN 구현
` Conv1-Relu1-Pool1-BN1-Conv2-Relu2-Pool2-BN2-Conv3-Relu3-Pool3-Conv4-Relu4-Pool4-BN3-Affine`
- Softmax로 확률 구현
- Adam으로 Optimizer 구현
- Batchsize는 1080, Epoch은 1000으로 훈련
- 훈련 파일을 ckpt 확장자로 저장 후 시험
### 진행 후
- 훈련 도중 기울기 손실이 없으나 학습률(η)을 0.001로 세팅한 것으로 인해 진행 부진
[NNNmk2trainsess.zip](https://github.com/roqhdehd502/Franklin-Detecter/files/3853956/NNNmk2trainsess.zip)
- 시험결과 정확도(Accuracy) 33.9%
`ACC 0.33877551020408164`
- 이 후 학습률을 높이거나 합성곱의 커널 속성을 수정하여 정확도를 높일 예정
